docs/sources/http_api/serviceaccount.md

```diff
@@ -0,0 +1,45 @@
++++
+title = "Service account HTTP API"
+description = "Grafana Service account HTTP API"
+keywords = ["grafana", "http", "documentation", "api", "serviceaccount"]
```

Comment:
```suggestion
keywords = ["grafana", "http", "documentation", "api", "service account"]
```

Reply:
Actually, I am not sure about change suggestion, given that the API path is `serviceaccount` and the documentation is `service account`. The former is harder to read.

Reply:
its a good point. I am not sure here if we want to separate the service account. Sorry to always point to google cloud, but here they separate service and account(s) in their documentation.

Maybe we should too?
https://console.cloud.google.com/iam-admin/serviceaccounts

Reply:
they also note plural and not singluar

Reply:
these keywords btw, can they be separate with a space?

Reply:
thx for the input